presto-docs/src/main/sphinx/release/release-0.253.rst

```diff
@@ -0,0 +1,28 @@
+=============
+Release 0.253
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix amplified `Cumulative User Memory` metric in web UI.
```

Comment:
Does single back quote do anything - as per my understanding it needs to be double back quotes

Reply:
single back tick make it italic

Reply:
I think single back ticks are for links. Italics should be between single asterisks.